examples/common/tracing/decoder/logging/ToCertificateString.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include <lib/support/Span.h>
+
+#include <stddef.h>
+
+namespace chip {
+namespace trace {
+namespace logging {
+
+const char * ToCertificateString(const chip::ByteSpan & data, char * buffer, size_t len);
```

Comment:
please add documentation of what these do.

Reply:
Done.